quad_ss_ctl:
  pt1_const: 0.7
  dead_zone: 0.0
  Td: 0.01
  n: 4.0
  m: 1.0
  l: 2.0
  A: [0.7791595631749835, -1.4961799981731592, 0.0, 0.0, 0.008864042317612508, 0.9922085930872637,
    0.0, 0.0, 0.35984106310602865, 77.74895247568584, 1.0, 0.0, 0.001223783679043055,
    0.38925494858856435, 0.01, 1.0]
  B: [0.00886404231761251, 4.6159794057238694E-5, 0.001223783679043055, 3.096346136515212E-6]
  C: [0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0]
  D: [0.0, 0.0]
  Kctl: [3.8679369757212125, 115.03784588554049, 0.1616413633557755, 0.318103702063354]
  Ki: -0.0026843471018401645
  Kobs: [-0.02639842113335235, 0.7426384376082065, 0.003943698327110571, -0.11765622383978393,
    1.1587112833214173, -16.767679897026042, 0.013061549667052114, 0.463300595541571]
  saturation: 0.3
  Tm: 0.2
  modelFaster: 10.0
  Kff: 0.02
